R/run-tests.sh

```diff
@@ -26,14 +26,30 @@ rm -f $LOGFILE
 SPARK_TESTING=1 $FWDIR/../bin/spark-submit --driver-java-options "-Dlog4j.configuration=file:$FWDIR/log4j.properties" --conf spark.hadoop.fs.default.name="file:///" $FWDIR/pkg/tests/run-all.R 2>&1 | tee -a $LOGFILE
 FAILED=$((PIPESTATUS[0]||$FAILED))
 
+# Also run the documentation tests for CRAN
+CRAN_CHECK_LOG_FILE=$FWDIR/cran-check.out
+rm -f $CRAN_CHECK_LOG_FILE
+CRAN_CHECK_OPTIONS="--no-tests --no-manual" $FWDIR/check-cran.sh 2>&1 | tee -a $CRAN_CHECK_LOG_FILE
+FAILED=$((PIPESTATUS[0]||$FAILED))
+
+HAS_CRAN_WARNING="$(grep -c WARNING $CRAN_CHECK_LOG_FILE)"
```

Comment:
should check for ERROR or NOTE too?

For ERROR, I guess that would go to exit code?
For NOTE, I think we should probably compare the note count. Perhaps then we just look for the line `Status: 2 NOTEs` at the end of the output?